integration_tests/poetry.lock

```diff
@@ -109,7 +109,7 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 [[package]]
 name = "cryptopy"
```

Comment:
there's no "cryptopy" -- afaik this was before the rename in https://github.com/crypto-org-chain/chainlibpy